test/worldwide/field_test.rb

```diff
@@ -188,5 +188,20 @@ class FieldTest < ActiveSupport::TestCase
         end
       end
     end
+
+    test "if an `_instructional` error key is submitted but that key does not exist, it will fall back to the original " \
```

Comment:
Let's also test the case "if an '_instructional' key suffix is omit, it will check for the suffixed key" 
ie the input error key is `contains_emojis`, but the error is defined under `contains_emojis_instructional` 

This is technically already covered with existing tests, but I find these tests helpful to understand the explicit desired behaviour.

Reply:
I would also be happy to see the new `_informative` error keys exercised, but this is not necessary.